pnpm-lock.yaml

```diff
@@ -6731,7 +6746,7 @@ packages:
     dependencies:
       universalify: 2.0.0
     optionalDependencies:
-      graceful-fs: 4.2.10
+      graceful-fs: 4.2.11
```

Comment:
This update looks correct when I check the package.json of the jsonfile module
It has the following entry:

```
 "optionalDependencies": {
    "graceful-fs": "^4.1.6"
  }, 
```

Reply:
It is best to also update the optional to be ^4.2.11

Reply:
Thanks for answering @donal-tobin-sap, but this has to be done manually if I understand correctly. 

Shouldn't this be doable by some `pnpm` command or maybe this is just a gap in `pnpm` as I do not see the usage of `^` in dependencies and `optionalDependencies`, but only in `peeerDependencies` section

Reply:
Yes, some manual work will be required.
Renovate (nor dependabot) do not do everything.